### PR TITLE
New feature: Challenge rounds have their own score titles

### PR DESCRIPTION
**Pull request by mridulnagpal**
#431

### CODE REVIEW

File: app/concepts/submission/views/table_row.erb [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <td><%= link_to participant.name, participant_path(participant) %></td>
   <td><span class="badge <%= grade_class %>"><%= entry.grading_status_cd %></span></td>
   <td><%= formatted_value(entry.score_display) %></td>
-  <% if entry.challenge.score_secondary_title? %>
+  <% unless secondary_score_title %>
```
Comment (nemaniarjun) [inline app/concepts/submission/views/table_row.erb]: This should be an IF statement right?

File: app/views/challenges/form/_details.html.erb [hunk 2]
```diff
@@ -166,7 +166,7 @@
 
   <div class="form-row">
     <div class="form-group col-6">
-      <label for="primaryScoreTitle">Primary score title</label>
+      <label for="primaryScoreTitle">Default primary score title </label>
```
Comment (nemaniarjun) [inline app/views/challenges/form/_details.html.erb]: Add a note saying that this can be changed in the rounds tab

File: db/schema.rb [hunk 3]
```diff
@@ -462,22 +464,6 @@
...
-    t.integer "model_id"
-    t.string "mailer_classname"
-    t.text "recipients"
-    t.text "options"
-    t.string "status_cd"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
-    t.string "email_preferences_token"
-    t.datetime "token_expiration_dttm"
-    t.integer "participant_id"
```
Comment (nemaniarjun) [inline db/schema.rb]: Unrelated changes to schema

File: app/concepts/submission/views/table_row.erb (hunk 1 above)
Comment (mridulnagpal) [reply app/concepts/submission/views/table_row.erb]: Resolved

File: db/schema.rb (hunk 3 above)
Comment (mridulnagpal) [reply db/schema.rb]: migrating leads to all those changes, removed schema changes from this PR

**Comment by nemaniarjun** [comment]
When you run migrations, the schema file changes, please add the schema file back into this PR but only for the parts that you changed. (that is changes to the part of the DB which you changed)

**Comment by nemaniarjun** [comment]
going to redo this once the cleanup is done properly
